src/components/Reports/Reports.test.js

```diff
@@ -46,13 +49,8 @@ describe('Reports page', () => {
       </DopplerIntlProvider>,
     );
 
-    await wait(() => getByText(fakeData[0].verified_date));
-
-    const verifiedDate = getByText(fakeData[0].verified_date);
-    const domain = getByText(fakeData[1].name);
-
-    expect(verifiedDate).toBeDefined();
```

Comment:
Why did you remove this expects? Shouldn't all tests have at least one expect?

Reply:
I removed because _getByText_ verify that the expected text exists, for example:

```html
<div>2018-01-01</div>
```

```js
const verifiedDate = getByText('2020-05-05'); // It breaks here.
expect(verifiedDate).toBeDefined(); // this line is never executed.
```

---

```html
<div>2018-01-01</div>
```

```js
const verifiedDate = getByText('2018-01-01'); // It pass, so verifiedDate is defined
expect(verifiedDate).toBeDefined(); // this line will pass
```